src/main.rs

```diff
@@ -10,11 +10,12 @@ async fn main() {
     setup_panic!();
 
     // make sure that the cursor re-appears when interrupting
-    tokio::spawn(async move {
-        tokio::signal::ctrl_c().await.unwrap();
-        let term = dialoguer::console::Term::stdout();
-        let _ = term.show_cursor();
-    });
+    // tokio::spawn(async move {
```

Comment:
we should not leave commented code.

Reply:
If this is a WIP, or a TODO, better put a clear comment section about what we want to do in the future.